File: server/plugin/pcdn/router/release.go
package router

import "github.com/gin-gonic/gin"

// ReleaseRouter agent 版本发布路由（admin 组）
type ReleaseRouter struct{}

// InitReleaseRouter 挂载版本路由
func (r *ReleaseRouter) InitReleaseRouter(group *gin.RouterGroup) {
	group.GET("release/list", releaseApi.GetReleaseList)
	group.GET("release/find", releaseApi.GetRelease)
	group.POST("release/create", releaseApi.CreateRelease)
	group.PUT("release/update", releaseApi.UpdateRelease)
	group.DELETE("release/delete", releaseApi.DeleteRelease)
}

File: web/src/plugin/pcdn/api/release.js
import service from '@/utils/request'

export const getReleaseList = (params) => service({ url: '/pcdn/admin/release/list', method: 'get', params })
export const getRelease = (params) => service({ url: '/pcdn/admin/release/find', method: 'get', params })
export const createRelease = (data) => service({ url: '/pcdn/admin/release/create', method: 'post', data })
export const updateRelease = (data) => service({ url: '/pcdn/admin/release/update', method: 'put', data })
export const deleteRelease = (params) => service({ url: '/pcdn/admin/release/delete', method: 'delete', params })
